go.mod

```diff
@@ -3,54 +3,66 @@ module github.com/GoogleCloudPlatform/ops-agent
 go 1.19
 
 require (
-	cloud.google.com/go/logging v1.4.2
-	cloud.google.com/go/monitoring v1.2.0
-	cloud.google.com/go/storage v1.18.2
+	cloud.google.com/go/compute v1.10.0
+	cloud.google.com/go/logging v1.5.0
+	cloud.google.com/go/monitoring v1.6.0
+	cloud.google.com/go/storage v1.26.0
+	github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric v0.32.8
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/cenkalti/backoff/v4 v4.1.2
-	github.com/go-playground/validator/v10 v10.9.0
-	github.com/goccy/go-yaml v1.9.4
-	github.com/google/go-cmp v0.5.6
+	github.com/cenkalti/backoff/v4 v4.1.3
+	github.com/go-playground/validator/v10 v10.11.1
+	github.com/goccy/go-yaml v1.9.5
+	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.3.0
 	github.com/hashicorp/go-multierror v1.1.0
 	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0
-	github.com/mitchellh/mapstructure v1.4.2
-	github.com/shirou/gopsutil v3.21.8+incompatible
-	go.uber.org/multierr v1.7.0
-	golang.org/x/sys v0.0.0-20211216021012-1d35b9e2eb4e
-	golang.org/x/text v0.3.6
-	google.golang.org/api v0.65.0
-	google.golang.org/genproto v0.0.0-20220118154757-00ab72f36ad5
-	google.golang.org/grpc v1.40.1
-	google.golang.org/protobuf v1.27.1
-	gopkg.in/yaml.v2 v2.2.3
+	github.com/mitchellh/mapstructure v1.5.0
+	github.com/shirou/gopsutil v3.21.11+incompatible
+	go.opentelemetry.io/contrib/detectors/gcp v1.10.0
+	go.opentelemetry.io/otel v1.10.0
+	go.opentelemetry.io/otel/metric v0.32.0
+	go.opentelemetry.io/otel/sdk v1.10.0
+	go.opentelemetry.io/otel/sdk/metric v0.31.0
```

Comment:
I would encourage you to update to v0.32.0, which has the new SDK implementation

Reply:
I tried to use the SDK version from `v0.32.0`, but the google cloud exporter in `opentelemetry-operations-go` still uses the previous implementation (e.g. the usage of `sdk/metric/sdkapi` module for the formatter). Is there a way to go around this dependencies and use the exporter with `v0.32.0` ? Or there is a need to first update `opentelemetry-operations-go` to use the new sdk in `v0.32.0` ?

Current usage : 
https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/main/exporter/metric/option.go#L18-L25

Reply:
Updated in last commit to use `v0.32.1`. Thanks for updating `opentelemetry-operations-go`.